### ISSUE TITLE
Needs multiple dependency overrides on flutter package

### ISSUE THREAD

**Issue by Prn-Ice** [ivoleitao/stash#34]
**Describe the bug**
I need to add the following dependency overrides to my flutter projects using stash 4.3.0

```yaml
dependency_overrides: 
  async: 2.8.2
  clock: 1.1.0
  meta: 1.7.0
  path: 1.8.1
```

**To Reproduce**

1. Create a new flutter project
2. Add stash and stash_objectbox
3. Run pub get

**Expected behavior**
Works without adding dependency_overrides

**Version**
4.3.0

**Comment by ivoleitao** [comment]
Humm it's not the first time this happens :-(. It's hard to understand which versions flutter uses of those packages so when I upgrade the dependencies this happens sometimes. I plan to add a flutter package in the future and that should help me catch those problems

 I will try to publish a 4.3.1 moving back that dependencies I don't need them upgraded.

**Comment by ivoleitao** [comment]
It's done I will publish it today

**Comment by ivoleitao** [comment]
Fixed, version 4.3.1 published
